src/NexusMods.App.UI/Controls/MiniGameWidget/MiniGameWidgetViewModel.cs

```diff
@@ -0,0 +1,75 @@
+using System.Reactive;
+using System.Reactive.Disposables;
+using System.Reactive.Linq;
+using Avalonia.Media.Imaging;
+using Microsoft.Extensions.Logging;
+using NexusMods.Abstractions.Diagnostics.Emitters;
+using NexusMods.Abstractions.GameLocators;
+using NexusMods.Abstractions.Games;
+using NexusMods.Abstractions.IO;
+using NexusMods.Abstractions.Library.Installers;
+using NexusMods.Abstractions.Loadouts.Synchronizers;
+using NexusMods.Abstractions.NexusWebApi.Types.V2;
+using NexusMods.Abstractions.Settings;
+using NexusMods.App.UI.Controls.GameWidget;
+using NexusMods.Icons;
+using NexusMods.MnemonicDB.Abstractions;
+using ReactiveUI;
+using ReactiveUI.Fody.Helpers;
+
+namespace NexusMods.App.UI.Controls.MiniGameWidget;
+
+public class MiniGameWidgetViewModel : AViewModel<IMiniGameWidgetViewModel>, IMiniGameWidgetViewModel
+{
+    private readonly ILogger<MiniGameWidgetViewModel> _logger;
+
+    public MiniGameWidgetViewModel(ILogger<MiniGameWidgetViewModel> logger, ISettingsManager settingsManager)
+    {
+        _logger = logger;
+
+        _image = this
+            .WhenAnyValue(vm => vm.Game)
+            // ReSharper disable once ConditionIsAlwaysTrueOrFalseAccordingToNullableAPIContract
```

Comment:
This seems odd.

Reply:
That was copied from the original GameWidget, if it's not needed then I'll strip it out of both